concent_api/api-integration-additional-verification-test.py

```diff
@@ -0,0 +1,367 @@
+        verify=False
+    )
+
+
+@count_fails
+def test_case_1_test_for_positive_case(cluster_consts, cluster_url, test_id):
+    current_time    = get_current_utc_timestamp()
+    (subtask_id, task_id) = get_task_id_and_subtask_id(test_id, 'existing_file')
+
+    file_content = task_id
+    file_size = len(file_content)
+    file_check_sum = 'sha1:' + hashlib.sha1(file_content.encode()).hexdigest()
+
+    ack_subtask_results_verify = api_request(
+        cluster_url,
+        'send',
+        REQUESTOR_PRIVATE_KEY,
+        CONCENT_PUBLIC_KEY,
+        get_subtask_results_verify(
+            task_id,
+            subtask_id,
+            current_time,
+            cluster_consts,
+            reason=message.tasks.SubtaskResultsRejected.REASON.VerificationNegative,
+            report_computed_task_size=file_size,
+            report_computed_task_package_hash=file_check_sum,
+            task_to_compute_size=file_size,
+            task_to_compute_package_hash=file_check_sum,
+        ),
+        headers = {
+            'Content-Type': 'application/octet-stream',
+        },
+        expected_status=200,
+        expected_message_type=message.concents.AckSubtaskResultsVerify.TYPE,
+        expected_content_type='application/octet-stream',
+    )
+
+    for file in ack_subtask_results_verify.file_transfer_token.files:
+        response = upload_file_to_storage_cluster(
+            file_content,
+            file['path'],
+            ack_subtask_results_verify.file_transfer_token
+        )
+        assert_condition(response.status_code, 200, 'File has not been stored on cluster')
+        print('\nUploaded file with task_id {}. Checksum of this file is {}, and size of this file is {}.\n'.format(
+            task_id,
+            file_check_sum,
+            file_size
+        ))
+
+    time.sleep(10)
+
+    api_request(
+        cluster_url,
+        'receive-out-of-band',
+        PROVIDER_PRIVATE_KEY,
+        CONCENT_PUBLIC_KEY,
+        create_client_auth_message(PROVIDER_PRIVATE_KEY, PROVIDER_PUBLIC_KEY, CONCENT_PUBLIC_KEY),
+        headers = {
+            'Content-Type': 'application/octet-stream',
+        },
+        expected_status=200,
+        expected_message_type=message.concents.SubtaskResultsSettled.TYPE,
+        expected_content_type='application/octet-stream',
+    )
+
+    api_request(
+        cluster_url,
+        'receive-out-of-band',
+        REQUESTOR_PRIVATE_KEY,
+        CONCENT_PUBLIC_KEY,
+        create_client_auth_message(REQUESTOR_PRIVATE_KEY, REQUESTOR_PUBLIC_KEY, CONCENT_PUBLIC_KEY),
+        headers = {
+            'Content-Type': 'application/octet-stream',
+        },
+        expected_status=200,
+        expected_message_type=message.concents.SubtaskResultsSettled.TYPE,
+        expected_content_type='application/octet-stream',
+    )
+
+
+@count_fails
+def test_case_2_test_for_invalid_reason(cluster_consts, cluster_url, test_id):
+    current_time    = get_current_utc_timestamp()
```

Comment:
This line is match with our coding style ?

Reply:
:+1: